api/public/cerbos/policy/v1/policy.proto

```diff
@@ -291,11 +291,16 @@ message TestResults {
   message Suite {
     string file = 1;
     string name = 2;
-    repeated Principal principals = 3;
+    repeated TestCase test_cases = 3;
```

Comment:
I think we should keep the previous `principals` field at number 3 and mark it as deprecated. That way we don't break the old versions horribly (i.e. there won't be a parsing error). They would just see an empty principal list.